src/Uno.UI/UI/Xaml/Media/Brush.Android.cs

```diff
@@ -89,33 +89,18 @@ internal static IDisposable AssignAndObserveBrush(Brush b, ColorSetterHandler co
 			}
 			else if (b is ImageBrush imageBrush && imageBrushCallback != null)
 			{
-				var disposables = new CompositeDisposable(5);
+				var disposables = new CompositeDisposable(6);
```

Comment:
Since you are making changes on Android, you should add a reference to the Android bug in your PR.

Reply:
Right! PR Title and description edited.